Source/AtomicEditor/Editors/ResourceEditor.cpp

```diff
@@ -65,6 +65,7 @@ class EditorTabLayout: public TBLayout
                 }
                 else
                 {
+                    editor_->SendEvent("EditorResourceCloseCanceled");
```

Comment:
yeah, I made a ResourceEditorEvents.h file in the first commit, but then when I saw https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Source/AtomicEditor/Editors/ResourceEditor.cpp#L165 , I thought that it's ok for small event to not define it with macro